main.py

```diff
@@ -156,38 +156,40 @@ async def nini(ctx):
         c += 1
         auth = m.author.name
         if auth not in all_losers:
-            all_losers[auth] = (0,0)
-        all_losers[auth] = (all_losers[auth][0] + 1, all_losers[auth][1])
+            all_losers[auth] = [0, 0, 0, 0]
+        all_losers[auth][NINI_MESSAGES] += 1
```

Comment:
Instead of defining and using NINI_* global variables, what do you think about using a python dictionary?

Maybe it's cleaner, and not *that* inefficient.